dbsync.py

```diff
@@ -486,11 +516,19 @@ def dbsync_init(mc, from_gpkg=True):
                                                  config.db_conn_info, config.db_schema_modified)
             summary_base = _compare_datasets("sqlite", "", gpkg_full_path, config.db_driver,
                                              config.db_conn_info, config.db_schema_base)
-            if len(summary_base):
-                raise DbSyncError("The db schemas already exist but 'base' schema is not synchronized with source GPKG")
-            elif len(summary_modified):
+            if len(summary_base) and not len(summary_modified):
+                # seems someone modified base schema manually - this should never happen!
+                raise DbSyncError("The db schemas already exist but 'base' schema is not synchronized with source GPKG "
+                                  "while modified schema is up to date")
+            elif len(summary_modified) and not len(summary_base):
                 print("Modified schema is not synchronised with source GPKG, please run pull/push commands to fix it")
                 return
+            elif len(summary_modified) and len(summary_base):
+                # here we are not sure whether base schema changes are safe (e.g. higher server version of project)
+                # or someone modified base schema manually which should not happen!
+                print("Both modified and base schemas are not synchronised with source GPKG, "
+                      "beware that bad things could happen")
+                return
```

Comment:
I think these two cases:
- `if len(summary_base) and not len(summary_modified):`
- `elif len(summary_modified) and len(summary_base):`
can be merged to one - simply just `if len(summary_base)` ... if base is not in sync with the GPKG, then it is a fatal error, and we don't even need to care about the "modified" schema (and have two different error messages)

So the logic probably can be simplified to:
1. if gpkg-base has changes -> fatal error (raise exception)
2. if gpkg-modified has changes -> print the summary of changes in 'modified' (and continue) and print that changes will be pushed in dbsync_push
    else -> just print that everything is awesome